internals/utils/quota_utils.go

```diff
@@ -185,19 +259,35 @@ func DoesSNSQuotaObjectExist(sns *ObjectContext) (bool, *ObjectContext, error) {
 	return false, nil, nil
 }
 
-// DoesSNSCrqExists returns true if a ClusterResourceQuota exists
-func DoesSNSCrqExists(sns *ObjectContext) bool {
-	snsCrq, err := NewObjectContext(sns.Ctx, sns.Client, types.NamespacedName{Name: sns.Object.GetName()}, &quotav1.ClusterResourceQuota{})
+// DoesSNSCRQExists returns true if a ClusterResourceQuota exists
+func DoesSNSCRQExists(sns *ObjectContext) (bool, *ObjectContext, error) {
```

Comment:
why use a bool and not just check if snsCrq != nil?

Reply:
It's more consistent with how these checks are done across the code (not saying it's the best way)

Reply:
I think it's creating unnecessary variables and we should consider changing it across the code. for now i'll approve.